.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-coverage
\ No newline at end of file
+coverage
+automation_testing/node_modules
+automation_testing/test-resutls
```

Comment:
@carol-bohorquez the line in the end says resuTLs, I believe this might cause issues in the future

Reply:
@mauriciodeleonc  I already fix this comment